common-lib/src/main/scala/com/gu/mediaservice/lib/elasticsearch/ElasticSearchExecutions.scala

```diff
@@ -39,7 +39,10 @@ trait ElasticSearchExecutions {
 
     result.failed.foreach { e =>
       val elapsed = System.currentTimeMillis() - start
-      val markers = MarkerContext(durationMarker(elapsed))
+      val markers = MarkerContext(appendEntries(Map(
+        "duration" -> elapsed,
+        "reason" -> e.getClass.getName
```

Comment:
This won't work as the error details are being swallowed by the `RuntimeException` on line 29

Reply:
(should refresh before commenting again)